goal_src/jak2/engine/debug/part-tester.gc

```diff
@@ -13,7 +17,7 @@
 (declare-file (debug))
 
 (defpartgroup group-part-tester
-  :id 127
+  :id 96
```

Comment:
Why was this changed?

Reply:
Main reason was that the `let` on line 71 was commented out and replaced with a `let` that uses id 96 instead for whatever reason. I don't think it matters much which id is used since neither seem to be used by the game.

Reply:
I think that was just them testing whatever 96 was at the time. IMO just keep it at 127, there's checks in place to make sure invalid part groups aren't tested anyway.